app/extensions/string_extensions.rb

```diff
@@ -483,16 +494,18 @@ def html_to_ascii
       gsub(/[ \t]+(\n|$)/, '\\1').    # remove superfluous trailing whitespace
       gsub(/\n+\Z/, "").              # remove superfluous newlines at end
       gsub(HTML_TAG_PATTERN, "").     # remove all <tags>
-      gsub(/^ +|[ \t]+$/, "").        # remove leading/trailing space on each line
-      gsub(/&(#\d+|[a-zA-Z]+);/) { HTML_SPECIAL_CHAR_EQUIVALENTS[Regexp.last_match(1)].to_s }.
-      html_safe                       # convert &xxx; and &#nnn; to ascii
+      gsub(/^ +|[ \t]+$/, "").        # remove leading/trailing sp on each line
+      gsub(                           # convert &xxx; and &#nnn; to ascii
+        /&(#\d+|[a-zA-Z]+);/
+      ) { HTML_SPECIAL_CHAR_EQUIVALENTS[Regexp.last_match(1)].to_s }.html_safe
```

Comment:
I'd isolate this in a separate function and only disable the cop for that function.